README.md

```diff
@@ -1 +1,174 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+is a first project to prepare the arena to showcase my future Portfolio as software developer. Here I will present my projects and professional skills.
+
+**[Portfolio]** is a starting point to present my future portfolio in a responsive way.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Standar code</summary>
+  <ul>
+    <li><a href="#">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[First approach to mobile responsive portfolio design]**
+- **[Introduction of my skills as professional developer]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Not available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Open on Visual Studio Code or any other code reader, and use an app as Live Server to review online.
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/HFG43/Portfolio_Mobile_First
+
+### Install
+
+Open index.html file with your Code Editor, and review the behaviour of the page under different sizes.
+
+### Usage
+
+To run the project, execute the following command:
+
+Run index.html on yor Web Browser to view changes.
+
+Example command on VSC: 
+Select your index.html file
+Run Open with Live Server.
+
+### Run tests
+
+To run tests, run the following command:
+
+No test available yet.
+
+### Deployment
+
+You can deploy this project using:
+
+No deployment available yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/HFG43)
+- Twitter: [@twitterhandle](https://twitter.com/HFG_43)
+- LinkedIn: [LinkedIn](linkedin.com/in/hernán-güemes-a440591b)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+**[dinamic_style_media_querys]**
+**[RWD_Responsive_Web_Design]**
+**[contact_form]**
+**[accesibility]**
+**[add_portfolio_snapshots]**
+**[deployment]**
+
```

Comment:
- [OPTIONAL] Good job adding **_future features_**🥇 I think it would be better if you add them like the way you added **_key features_** What do you think? 🤔

Future features | Key features
---| ---|
![image](https://user-images.githubusercontent.com/106119105/234744655-385604f3-81ad-460f-a4d9-0eafe4edccd4.png) | ![image](https://user-images.githubusercontent.com/106119105/234744703-1a457aa3-8d6f-4a66-8ba7-a3d3b0ff64a5.png)